docs/operations/using-rollong-upgrades.md

```diff
@@ -0,0 +1,19 @@
+---
+linkTitle: Using Rolling Upgrades Feature 
+---
+
+# Enabling Rolling Upgrades 
+
+To prevent your persistent storage volumes from becoming unhealthy during orchestrator update you need to enable rolling upgrades. Note, that if your volume doesn't have any replicas the rolling upgrades will be disabled by default. 
+
+# Procedure
+
+Enable both node manager and the upgrade guard by adding the following to the StorageOSCluster spec: 
+
+```
+ nodeManagerFeatures:
```

Comment:
There is a page on the opreator examples that show different areas of the spec. of the CR, I think it might be good to add  this section there as an example.